encoding_test.go

```diff
@@ -86,3 +86,33 @@ func TestDecodeBytes_invalidVarint(t *testing.T) {
 	_, _, err := decodeBytes([]byte{0xff})
 	require.Error(t, err)
 }
+
+var versions = []int64{
+	0,
+	1,
+	100,
+	127,
+	128,
+	1 << 29,
+	-0,
+	-1,
+	-100,
+	-127,
+	-128,
+	-1 << 29,
+}
+
+var sink interface{}
```

Comment:
We should move these inside the benchmark function scope.

Reply:
@erikgrinaker versions yes, we can move inside, but sink we have to leave as a global to ensure the benchmark runs successfully. I've added a comment to sink to indicate why, and then moved versions inside. Please take another look, and thank you for the review!